tests/test_core.py

```diff
@@ -3,82 +3,64 @@
 import numpy as np
 from turbopy.core import Grid
 
-def test_grid():
-    """Test initialization of the Grid class"""
-    n_grid = 8
-    grid_conf = {"N": n_grid,
+#Grid class test methods
+@pytest.fixture(name='conf')
+def grid_conf():
+    """Pytest fixture for grid configuration dictionary"""
+    grid_dict = {"N": 8,
                  "r_min": 0,
                  "r_max": 0.1}
-    grid = Grid(grid_conf)
-    assert grid.grid_data == grid_conf
+    return grid_dict
```

Comment:
Looking good. I think if you write
```suggestion
    return grid
```
here instead of returning `grid_dict`, that will save you a few more lines.